index.html

```diff
@@ -3,30 +3,120 @@
     <head>
         <meta charset="UTF-8">
         <link href=".//assets/css/styles.css" rel="stylesheet">
-        <title>EuroTrack</title>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+        <title>Track This!</title>
     </head>
 
     <body>
         <header>
-            <div class="hero"></div>
-            <div>
```

Comment:
the below code is the working carousel

<div id="demo" class="carousel slide" data-bs-ride="carousel">
              <div class="carousel-indicators">
                <button type="button" data-bs-target="#demo" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
                <button type="button" data-bs-target="#demo" data-bs-slide-to="1" aria-label="Slide 2"></button>
                <button type="button" data-bs-target="#demo" data-bs-slide-to="2" aria-label="Slide 2"></button>
                <button type="button" data-bs-target="#demo" data-bs-slide-to="3" aria-label="Slide 4"></button>
                <button type="button" data-bs-target="#demo" data-bs-slide-to="4" aria-label="Slide 5"></button>
              </div>
              <div class="carousel-inner">
                <div class="carousel-item active">
                  <img src="./assets/images/brazil.jpg" class="d-block w-100" alt="Brazil Panorama">
                </div>
                <div class="carousel-item">
                  <img src="./assets/images/china.jpg" class="d-block w-100" alt="Great Wall">
                </div>
                <div class="carousel-item">
                  <img src="./assets/images/india.jpg" class="d-block w-100" alt="Taj Mahal">
                </div>
                <div class="carousel-item">
                  <img src="./assets/images/mexico.jpg" class="d-block w-100" alt="Mexico">
                </div>
                <div class="carousel-item">
                  <img src="./assets/images/usa.jpg" class="d-block w-100" alt="Statue of Liberty">
                </div>
              </div>
                <button class="carousel-control-prev" type="button" data-bs-target="#demo" data-bs-slide="prev">
                  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                  <span class="visually-hidden">Previous</span>
                </button>
                <button class="carousel-control-next" type="button" data-bs-target="#demo" data-bs-slide="next">
                  <span class="carousel-control-next-icon" aria-hidden="true"></span>
                  <span class="visually-hidden">Next</span>
              </button>
            </div>
          </div>